### ISSUE TITLE
Significant Drop in ATE and Rendering Performance on 7-Scenes

### ISSUE THREAD

**Issue by EH-HU** [GradientSpaces/WildGS-SLAM#35]
I tested the code on the 7-Scenes dataset and observed a significant drop in both ATE and rendering performance. However, I’m not sure what is causing this degradation. Could you help me understand why this might be happening?
Since there is no dynamic object, the uncertainty was not used.

**Comment by Jianhao-zheng** [comment]
Hi, I have run on chess sequence and get 0.037m ATE. The rendered images also look pretty well. I used the same hyperparameter as TUM RGB-D. Either activating or deactivating the uncertainty mask for both tracking and mapping gets the same result. Please check your implementation and the intrinsic parameters for this scene. FYI, I have added the dataloader and config for 7-scene in this [commit](https://github.com/GradientSpaces/WildGS-SLAM/commit/498c768911ce6a4a249ff14db603a0b37d1d66a0)

**Comment by EH-HU** [comment]
Could you please explain what kind of impact this depth_scale parameter has on the system?
mapping:
  Calibration:
    depth_scale: 5000.0
